atlasdb-workload-server-distribution/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM adoptopenjdk/openjdk11:jre-11.0.9_11.1-alpine
+MAINTAINER AtlasDB Team
+
+# Bash is useful for monitoring; curl is used by our healthchecks.
+RUN apk update && apk add bash curl
```

Comment:
nit: Technically the comment is no longer true I think? This was probably from the timelock-server distribution and `curl` was only for the container healthcheck here. Though I think we should keep both, and we can just say they're both useful for monitoring.